example/extensions/lib_custom_op/test_transposecsr.py

```diff
@@ -54,25 +54,25 @@
 print("indices:", c.indices.asnumpy())
 print("indptr:", c.indptr.asnumpy())
 
-print("--------start symbolic compute--------")
+print("--------start Gluon compute--------")
 d = mx.sym.Variable('d')
```

Comment:
should this sym also get deprecated then?

Reply:
No, we still use Symbol class, but we removed all the usages like `bind`